### PR TITLE
Log KMS decrypt errors in Asherah Go

### PR DESCRIPTION
**Pull request by dalibor**
Useful to print the error before `continue`. I had to build `asherah-cobhan` binaries with patched Asherah Go to be able to detect the issue documented in https://github.com/godaddy/asherah-cobhan/pull/39.

### CODE REVIEW

**Comment by jgowdy** [comment]
Logo
**Checkmarx AST – Scan Summary & Details** – [46fe8788\-1e88\-47eb\-9abd\-8109d730efe1](https://ast.checkmarx.net/projects/df34b7f2-6257-4bd2-a1e0-ecf40b2777d3/scans?id=46fe8788-1e88-47eb-9abd-8109d730efe1&branch=print_kms_errors)
### No New Or Fixed Issues Found

File: go/appencryption/pkg/kms/aws.go [hunk 1]
```diff
@@ -281,11 +281,13 @@ func (m *AWSKMS) DecryptKey(ctx context.Context, keyBytes []byte) ([]byte, error
 			decryptKeyTimer.UpdateSince(start)
 
 			if err != nil {
+				log.Printf("error kms decrypt: %s", err)
```
Comment (aka-bo) [inline go/appencryption/pkg/kms/aws.go:284]: Need to use [`Debugf`](https://github.com/godaddy/asherah/blob/a6c6eacc25a5b6a020b9f8e6242c090ced4ff7a6/go/appencryption/pkg/log/log.go#L18) from the internal `log` package to ensure these are only logged when debugging is enabled.

File: go/appencryption/pkg/kms/aws.go [hunk 2]
```diff
@@ -281,11 +281,13 @@ func (m *AWSKMS) DecryptKey(ctx context.Context, keyBytes []byte) ([]byte, error
...
 				continue
 			}
 
 			decryptedKeyBytes, err := m.Crypto.Decrypt(en.EncryptedKey, output.Plaintext)
 			if err != nil {
+				log.Printf("error crypto decrypt: %s", err)
```
Comment (aka-bo) [inline go/appencryption/pkg/kms/aws.go:290]: same here

**Review by aka-bo — CHANGES_REQUESTED** [review]

**Review by aka-bo — APPROVED** [review]
LGTM

**Pull request merged**